### PR TITLE
LIT 3143 3972 4336

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
I tried to complete and disambiguate the genealogies related to Takla Hazmanot.

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! Everything that is there seems fine to me, and is very helpful. But I would be grateful if you could include an abstract in each record which very briefly describes the work in question and , if relevant, its relation/disambiguation from the other works. I think that this is one of the cases where paragraphic information is appropriate.

**Comment by DenisNosnitsin1970** [comment]
I have tried.

**Comment by thea-m** [comment]
Here, in this PR? I don't see an abstract in the records.

**Comment by DenisNosnitsin1970** [comment]
Now. No idea why it got stuck in the pipe.

File: 3001-4000/LIT3143Ledata.xml [hunk 1]
```diff
@@ -39,20 +35,62 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             <keywords scheme="#ethioauthlist">
                <term key="HistoryAndHistoriography"/>
+                <term key="ChristianLiterature"/>
+                <term key="Hagiography"/>
             </keywords>
+
          </textClass>
+         <abstract>
+             <p>The monastic genealogy of <persName ref="PRS9162TaklaHa">St Takla Hāymānot</persName> that shows
+             his spiritual descendance from <persName ref="PRS1385Adam">St Anthony</persName> and tells about his disciples
```
Comment (thea-m) [inline 3001-4000/LIT3143Ledata.xml:51]: Adam or Anthony?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! One minor question.

File: 3001-4000/LIT3143Ledata.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 3001-4000/LIT3143Ledata.xml]: Anthony.

**Review by thea-m — APPROVED** [review]

**Pull request merged**